.github/workflows/post-release.yml

```diff
@@ -101,4 +110,12 @@ jobs:
             -a tag=latest \
             buildpacksio/lifecycle:latest@${MANIFEST_SHA}
           cosign verify -key cosign.pub -a tag=latest buildpacksio/lifecycle:latest
+
           cosign attach sbom -sbom ./*-bom.cdx.json -type cyclonedx buildpacksio/lifecycle:latest
+          SBOM_SHA=$(cat *-checksums.txt | grep bom.cdx.json | awk '{print $1}')
```

Comment:
Same here.

Reply:
Ohh... I was intending to put @sha256 here: https://github.com/buildpacks/lifecycle/pull/701/files#diff-dfb63fa587b638fcf581007b525fdfb26835766b2a2be8840d19c6422f7d0ae6R120 but I forgot!

Reply:
It's also included the checksums.txt file in the release (in case someone would like to consume the json file that is also attached to the release).

Reply:
Added in 707398c

Reply:
See https://github.com/buildpacks/lifecycle/pull/701#discussion_r703685234, removed the SBOM sha for now.